packages/package-json-lint/README.md

```diff
@@ -0,0 +1,26 @@
+# package.json Linter
+
+[![NPM version](https://badgen.net/npm/v/@cerner/package-json-lint)](https://www.npmjs.org/package/@cerner/package-json-lint)
+[![Build Status](https://badgen.net/travis/cerner/terra-toolkit)](https://travis-ci.com/cerner/terra-toolkit)
+
+Linter for package.json files
+
+## Versioning
+
+package-json-lint is considered to be stable and will follow [SemVer](http://semver.org/) for versioning.
+
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs, related issues, and PRs for more information.
```

Comment:
Can this be removed since it's describe at the root level?

Reply:
I was mimicking existing packages. As I think about it, it wouldn't show up in the npm page if we didn't include that (if that matters): e.g. https://www.npmjs.com/package/@cerner/terra-cli